tap-gui/tap-gui-rbac/setup-tap-gui-rbac-eks.md

```diff
@@ -0,0 +1,157 @@
+# Enable Authorization on Remote EKS Clusters in Tanzu Application Platform GUI
+
+The process for adding access-controlled visibility for a remote EKS cluster is the following:
+
+1. Set up the OIDC provider (pre-requisite)
+2. Configure the EKS cluster with the OIDC provider
+3. Configure the Tanzu Application Platform GUI to view the remote EKS cluster
+4. Upgrade the Tanzu Application GUI package
+
+Once these steps are complete, you can view your runtime resources on a remote EKS cluster in Tanzu Application Platform GUI. For more detail, please refer to [View Runtime Resources on Remote Clusters in Tanzu Application Platform GUI](./view-resouces-rbac.md).
+
+## <a id="set-up-oidc-provider"></a> Set up the OIDC provider
+
+To enable RBAC visibility of remote EKS clusters, you need to set up the OIDC provider. You can see the list of supported OIDC providers in the [Setting up a Tanzu Application Platform GUI authentication provider](./../auth.md) section.
+
+Tanzu Application Platform GUI supports multiple OIDC providers. For the purposes of this guide, we shall use Auth0 as an example.
+
+1. Log in to the Auth0 dashboard
+2. Navigate to Applications
+3. Create an Application
+   - Name: `TAP-GUI` (or your custom app name)
+   - Application type: `Single Page Web Application`
+1. Click on the `Settings` tab
+2. Add under Application URIs > Allowed Callback URLs : `http://tap-gui.INGRESS-DOMAIN/api/auth/auth0/handler/frame`
+   
+    Where:
+
+    - `INGRESS-DOMAIN` is the domain you have chosen for your Tanzu Application Platform GUI in [Installing the Tanzu Application Platform package and profiles](install.md.hbs).
+
+
+1. Click `Save Changes`
```

Comment:
This numbering might be a bit wonky? I'm assuming you want these numbers to keep incrementing 1-6. If so, I think you wanna use `1.` at the beginning of every line, or else explicitly write 1., 2., ... 5., 6.